app/assets/stylesheets/ama_layout/layout_components/breadcrumbs.scss

```diff
@@ -5,4 +5,16 @@
     display: inline-block;
     font-size: 0.75rem;
   }
+
+  a.disabled{
```

Comment:
Sure - but foundation uses the `disabled` class for color I believe.

See; http://foundation.zurb.com/sites/docs/breadcrumbs.html